src/lib.rs

```diff
@@ -739,6 +739,7 @@ pub use http;
 pub use hyper::Server;
 pub use tower_http::add_extension::{AddExtension, AddExtensionLayer};
 
+#[doc(no_inline)]
 pub use crate::json::Json;
```

Comment:
Sorry if it wasn't clear, but this should be the canonical definition without the attribute; the re-exports in `response.rs` and `extract.rs` should re-export this one (rather than the original definition) with the `doc(no_inline)` attribute.

Reply:
So all re-export types should be added with `#[doc(no_inline)]`?